app.py

```diff
@@ -46,7 +47,8 @@
        "reaper_sleep_secs": 30,                        # How long should the reaper process sleep in between runs?
        "debug": 0,                                     # Set debug mode
        "narrenv": dict(),                              # Dictionary of env name/val to be passed to narratives at startup
-       "num_prespawn": 5}                              # How many prespawned narratives should be maintained? Checked at startup and reapee runs
+       "num_prespawn": 5,                              # How many prespawned narratives should be maintained? Checked at startup and reapee runs
+       "status_users": ["sychan", "kkeller", "jsfillman", "scanon", "bsadhkin"]}  # users with full narratve_status privs, env var use comma separated usernames
```

Comment:
Maybe should call AUTH here and look for a NARRATIVE_TRAEFIKER_ADMIN role with a token a.k.a check auth role instead of a comma separated list.

Reply:
Good idea. Let me look at that API.

Reply:
@sychan an example is here

https://github.com/kbase/catalog/blob/master/lib/biokbase/catalog/controller.py#L1214-L1221

Reply:
I'm ambivalent about using an auth role, mostly because it slightly complicates the check_mk rule (we need to add a header with the token).  But that's a minor quibble.

Reply:
Actually I misread the code.  Yeah, we should definitely use a role instead of a hardcoded user list.

Reply:
I will put this into the next PR for this branch. I want to get the basic functionality into a testable state first.